plugins/TestRunner/templates/matomo-tests.yml.twig

```diff
@@ -220,6 +222,7 @@ jobs:
         uses: matomo-org/github-action-tests@main
         with:
 {% if plugin %}
+          testomatio: ${{ '{{secrets.TESTOMATIO}}' }}
```

Comment:
@achakko FYI new plugins in future, will have this line automatically from next release

Reply:
Awesome, thanks much @AltamashShaikh !